### PR TITLE
Added support for fzy

### PR DESCRIPTION
**Pull request by nwaywood**
As discussed here https://github.com/jarun/nnn/pull/502, added support for `fzy` as well.

Also renamed the plugin from `fzfz` to `fzz` since it's not specific to `fzf` anymore.

### CODE REVIEW

File: plugins/fzz [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#
+# Shell: POSIX compliant
+# Author: Nick Waywood
+
+. "$(dirname "$0")"/.nnn-plugin-helper
+
+if [ "$(cmd_exists fzf)" -eq "0" ]; then
+    if [ -f "$HOME/.z" ]; then
+        # I read the data from z's file instead of calling the z command so that the data doesn't need to be processed twice
+        sel=$(awk -F "|" '{print $1}' "$HOME/.z" | fzf | awk '{$1=$1};1')
```
Comment (jarun) [inline plugins/fzz:13]: Can we check the existence of the utility first and the use `$fuzzy` like we do in this plugin?

https://github.com/jarun/nnn/blob/master/plugins/fzopen

That would avoid cod repetition.

Comment (nwaywood) [reply plugins/fzz]: @jarun done

**Comment by jarun** [comment]
Thank you so much for your patience and fixing the plugin!

**Comment by jarun** [comment]
I force-pushed to remove the earlier fzfz file and entry (with your credits intact). You may need to re-fork to be in sync with master.

**Comment by step-** [comment]
@nwaywood : please consider supporting z's environment variable [$\_Z\_DATA](https://github.com/rupa/z/blob/bbec3cb6af844e3f6d95bd7f28435abe0b5d21af/z.sh#L12). Thank you.
Suggested patch:
```diff
--- fzz	2020-04-17 14:12:28.605178360 +0000
+++ new	2020-04-17 14:16:06.309235322 +0000
@@ -15,11 +15,12 @@
     exit 1
 fi
 
-if [ -f "$HOME/.z" ]; then
+datafile="${_Z_DATA:-$HOME/.z}"
+if [ -f "$datafile" ]; then
     # I read the data from z's file instead of calling the z command so that the data doesn't need to be processed twice
-    sel=$(awk -F "|" '{print $1}' "$HOME/.z" | "$fuzzy" | awk '{$1=$1};1')
+    sel=$(awk -F "|" '{print $1}' "$datafile" | "$fuzzy" | awk '{$1=$1};1')
     # NOTE: Uncomment this line and comment out the line above if you want to see the weightings of the dir's in the fzf/fzy pane
-    # sel=$(awk -F "|" '{printf "%s %s\n", $2, $1}' "$HOME/.z" | "$fuzzy" | sed 's/^[0-9,.]* *//' | awk '{$1=$1};1')
+    # sel=$(awk -F "|" '{printf "%s %s\n", $2, $1}' "$datafile" | "$fuzzy" | sed 's/^[0-9,.]* *//' | awk '{$1=$1};1')
 else
     exit 1
 fi
```

**Comment by jarun** [comment]
@step- you can raise the PR as well.

**Pull request merged**